handlers/InitiateSalesforceSessionHandler.ts

```diff
@@ -152,11 +153,11 @@ export class InitiateSalesforceSession {
 									if (queuePosition === 0) {
 										// User Queue Position = 0
 										const queuePosMessage = LANoQueueMessage.replace(/%s/g, queuePosition);
-										await sendLCMessage(this.modify, this.data.room, queuePosMessage, this.data.agent, true);
+										await sendLCMessage(this.modify, this.data.room, queuePosMessage, this.data.agent, postChatUrl, true);
```

Comment:
@bhardwajaditya Instead of passing `postChatUrl` in every function, it's a good idea to save `postChatUrl` in the room custom fields and use it in this function. We will need to pass `IRead` to get the latest custom fields. So instead of `postChatUrl`, we will pass `IRead` and I think it's a cleaner way to do it generically than making it specific to postChatUrl.

I have tested it in my local I will push the code in some time. Can you please verify whenever you get chance please? Thanks! 😄